webroot/js/participantSummary.js

```diff
@@ -10,6 +20,37 @@ function loadAllSitesCallback(result) {
     siteList = JSON.parse(result).map((item) => { return { name: item.name, id: item.id, clientTypes: item.clientTypes } });
 };
 
+function setSearchValue(searchText) {
+    document.querySelector('#site-search').value = searchText;
+    $(resultsElement).hide();
+    $('#doSearch').click();
+}
+
+function searchSitesAutocomplete(e) {
```

Comment:
do we need the `e` parameter? it's unused

Reply:
we don't need it but I figured it's an okay reminder that the button that was pressed is available for use in case we want to do something like catch the enter key at some point.